### DISCUSSION TITLE
unpaid block found at pool

### DISCUSSION THREAD

**Discussion by bluehawana** [Chia-Network/chia-blockchain#11392] [Support]
hi，
I have a found block on the pool but did not get paid from Chia，would you pls help me with this issue？Thanks a lot！

BR
bluehawana

**Comment by Scrooghy** [comment]
Contact pool support. Discord

**Reply by bluehawana** [reply]
Yes, they told me I need to contact Chia.

**Reply by loppefaaret** [reply]
Hi - could you give us a bit more to go on ?
what is your current "Farmer rewards target address" that has not gotten its farming reward of 0.25 XCH from a block win ?
what block do you believe to have won, and not getting rewarded for ?
